migration/identity-migration/src/main/java/io/camunda/migration/identity/config/oidc/OidcProperties.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.migration.identity.config.oidc;
+
+public class OidcProperties {
+
+  private Audience audience = new Audience();
+
+  public Audience getAudience() {
+    return audience;
+  }
+
+  public void setAudience(final Audience audience) {
+    this.audience = audience;
+  }
+
+  public static class Audience {
```

Comment:
🔧 as it's multiple, I would prefer plural
```suggestion
  public static class Audiences {
```